CribblyBackend/Services/TeamService.cs

```diff
@@ -0,0 +1,86 @@
+using System.Data;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using Dapper;
+
+namespace CribblyBackend.Services
+{
+    public interface ITeamService
+    {
+        Task<Team> GetById(int Id);
+        void Update(Team Team);
+        Task Create(Team Team);
+        void Delete(Team Team);
+    }
+    public class TeamService : ITeamService
+    {
+        IDbConnection connection;
+        public TeamService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public async Task Create(Team team)
+        {
+            if(team.Players.Count < 2)
+            {
+                throw new System.NullReferenceException("A Team must not have less than two players");
```

Comment:
Perhaps we can find a more suitable exception type; or just throw the generic `Exception`

Reply:
ArgumentNull? Otherwise we can do a general Exception

Reply:
Gonna go with a general Exception and merge